Source/Blazorise/JsOptions/ColorPickerJsOptions.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+
+namespace Blazorise;
+
+public record ColorPickerJsOptions(
+    string Default,
+    string[] Palette,
+    bool ShowPalette,
+    bool HideAfterPaletteSelect,
+    bool ShowClearButton,
+    bool ShowCancelButton,
+    bool ShowOpacitySlider,
+    bool ShowHueSlider,
+    bool ShowInputField,
+    bool Disabled,
+    bool ReadOnly,
+    IReadOnlyDictionary<string, string> Localization,
+    string ColorPreviewElementSelector,
+    string ColorValueElementSelector
+    );
+
+public record OptionChange(bool Changed, bool Value);
+public record PaletteChange(bool Changed, string[] Value);
```

Comment:
Maybe `OptionChange` can be a generic object so that we don't need to have new this `PaletteChange` or similar? Just `OptionChange<string>`, `OptionChange<string[]>`, etc.

Reply:
Yes!